docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,135 @@
+********************************
+Configuring Grafana and InfluxDB
+********************************
+
+InfluxDB
+========
+
+The Sawtooth team provides Docker files for Grafana and InfluxDB for reporting statistics and monitoring network health.
+
+First we will build the InfluxDB docker container
```

Comment:
This is a frill, but ...  It would be nice to have numbered steps in each section (for clarity and to help us skimmers to not miss a step).  

Basically, you start a line with:
      #. Text... 

Then indent all following lines exactly 3 spaces, including any ..code-block:: or ..note:: chunks. Careful:  If you indent 4 spaces, the first line is bolded.